Makefile

```diff
@@ -1,12 +1,13 @@
 PREFIX?=/usr/local
 PKG_CONFIG?=pkg-config
 INSTALL=install
+KVERSION = $(shell uname -r)
 PYTEST=pytest-3
-CFLAGS+=-std=gnu89 -O2 -g -MMD -Wall				\
```

Comment:
Let's leave -g on in release builds - nothing's worse than hitting a bug that you just need a backtrace for and gdb doesn't give you anything.

Reply:
agree.  Maybe it was not a genious idea after all ...